src/Representation/Client.php

```diff
@@ -113,7 +112,6 @@ public function __construct(
         protected ?string $name = null,
         protected ?int $nodeReRegistrationTimeout = null,
         protected ?int $notBefore = null,
-        protected ?bool $oauth2DeviceAuthorizationGrantEnabled = null,
```

Comment:
In order to maintain compatibility with previous Keycloak versions (in this case `< 20.0.1`) you can use the attributes `Fschmtt\Attribute\Since` and `Fschmtt\Attribute\Until`:

```php
use Fschmtt\Attribute\Until;

...


#[Until('20.0.1')]
protected ?bool $oauth2DeviceAuthorizationGrantEnabled = null,
```

The `PropertyFilter` will then remove this property before sending the request.

Reply:
While adding the `Clients` resource (https://github.com/fschmtt/keycloak-rest-api-client-php/pull/46) I figured out that the property is not existing since at lear Keycloak 13.0.0 (which is the oldest version in the test matrix).

Sorry for the confusion, I initially thought it has been removed just recently 🙈